packages/flutter/lib/src/painting/image_resolution.dart

```diff
@@ -284,18 +285,29 @@ class AssetImage extends AssetBundleImageProvider {
     Completer<AssetBundleImageKey>? completer;
     Future<AssetBundleImageKey>? result;
 
-    chosenBundle.loadStructuredData<Map<String, List<String>>?>(_kAssetManifestFileName, manifestParser).then<void>(
-      (Map<String, List<String>>? manifest) {
-        final String chosenName = _chooseVariant(
+    // TODO(andrewkolos): Once google3 and google-fonts-flutter are migrated
+    // away from using AssetManifest.json, remove all references to it.
+    // See https://github.com/flutter/flutter/issues/114913.
+    chosenBundle
```

Comment:
Based on my read of the presubmit failures, it looks like we're not catching the error that propagates when loadStructuredBinaryData fails to lookup an asset, causing an unhandled async exception.